### PR TITLE
spatial_ref_sys should be replicated table in order to adapt gp6, oth…

### PR DESCRIPTION
**Pull request by lambgong**
…erwise we will get 'function cannot execute on a QE slice because it accesses relation' error

### CODE REVIEW

**Comment by orhankislal** [comment]
This is a good idea but we have to make sure it is only run on gpdb6. We currently have the `libsrid.c` file to work around this issue for various built-in postgis functions. To test this change, we should revert all of the changes related to this table first.

**Comment by lambgong** [comment]
Please let me know if you slove this issue

File: postgis/build/postgis-2.5.2/postgis/postgis.sql.in [hunk 1]
```diff
@@ -1924,7 +1924,7 @@ CREATE TABLE spatial_ref_sys (
 	 auth_srid integer,
 	 srtext varchar(2048),
 	 proj4text varchar(2048)
-);
+) DISTRIBUTED REPLICATED;
```
Comment (zhuyuting921026) [inline postgis/build/postgis-2.5.2/postgis/postgis.sql.in:1928]: TEST

Comment (zhuyuting921026) [inline postgis/build/postgis-2.5.2/postgis/postgis.sql.in:1928]: TEST

File: postgis/build/postgis-2.5.2/postgis/postgis.sql.in [hunk 2]
```diff
@@ -1924,7 +1924,7 @@ CREATE TABLE spatial_ref_sys (
 	 auth_srid integer,
 	 srtext varchar(2048),
```
Comment (zhuyuting921026) [inline postgis/build/postgis-2.5.2/postgis/postgis.sql.in:1925]: 大大

**Comment by orhankislal** [comment]
Closing this PR since https://github.com/greenplum-db/geospatial/pull/61 handles this issue.

**Pull request closed without merging**